www/js/theme_editor.js

```
@@ -81,31 +88,26 @@ const imageInput = () =>
           // if (!themesWithCustomBg.includes(curTheme)) {
           store.setUserPref('app.theme', themesWithCustomBg[0]);
           // }
-          try {
-            const userBackgroundsPath = `${userDataPath}/user_backgrounds`;
+          const userBackgroundsPath = `${userDataPath}/user_backgrounds`;
 
+          try {
```

added another try so we don't swallow all errors in one block.